src/version.php

```diff
@@ -3,6 +3,6 @@
     define('WOVN_PHP_NAME', 'WOVN.php');
 }
 if (!defined('WOVN_PHP_VERSION')) {
```

Comment:
Minor bump (0.2.0). We definitely need to move wovn.php to 1.0.0 soon as well though.

Reply:
I changed 0.2.0 .
I will bump up to 1.0.0 soon after relase this PR.